SimPEG/data_misfit.py

```diff
@@ -154,43 +199,91 @@ def W(self, value):
         self._W = value
 
     def residual(self, m, f=None):
+        r"""Computes the data residual vector for a given model.
+
+        Where :math:`\mathbf{d}_obs` is the observed data vector and :math:`\mathbf{d}_{pred}`
```

Comment:
Minor correction in LaTeX equation:
```suggestion
        Where :math:`\mathbf{d}_\text{obs}` is the observed data vector and :math:`\mathbf{d}_\text{pred}`
```